doc/man3/EVP_MAC.pod

```diff
@@ -145,6 +145,9 @@ the key.  If I<key> is NULL, the key must be set via I<params> either
 as part of this call or separately using EVP_MAC_CTX_set_params().
 Providing non-NULL I<params> to this function is equivalent to calling
 EVP_MAC_CTX_set_params() with those I<params> for the same I<ctx> beforehand.
+Note: There are additional requirements for some MAC algorithms during
+re-initalization (i.e. calling EVP_MAC_init() on an EVP_MAC after EVP_MAC_final()
```

Comment:
nit: unclosed (

Reply:
I'm replacing the parenthesis, but the sentence does seem to start with a capital (There). I believe the i.e. should remain in lower case, as it is in the middle of a sentence.